sys/arm64/arm64/machdep.c

```diff
@@ -1410,6 +1448,14 @@ DB_SHOW_COMMAND(specialregs, db_show_spregs)
 #define	PRINT_REG(reg)	\
     db_printf(__STRING(reg) " = %#016lx\n", READ_SPECIALREG(reg))
 
+#if __has_feature(capabilities)
+#define	PRINT_REG_CAP(reg)						\
+do {									\
+    void * __capability _tmp = (void * __capability)READ_SPECIALREG_CAP(reg); \
+    db_printf(__STRING(reg) " = %#lp\n", _tmp);				\
```

Comment:
```suggestion
    db_printf(__STRING(reg) " = %#.16lp\n", _tmp);				\
```
to match the integer registers